.tidyrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "importSort": "ide",
+  "importWrap": "auto",
+  "indent": 2,
+  "operatorsFile": ".tidyoperators",
+  "ribbon": 1,
+  "typeArrowPlacement": "first",
+  "unicode": "never",
+  "width": 80
```

Comment:
Can we do 120 as in Haskell? 80 columns is almost a mockery...

Reply:
I don't share that preference personally... I prefer vertical spread over horizontal spread. 80 col is pretty standard, and is the default in `prettier`. It makes it nice for stacking panes horizontally, which is a workflow I use quite often. I'd rather keep it at 80 if that's alright.

Reply:
Keep in mind - the indent size is 2 spaces and purs-tidy does not horizontally align blocks as we do for Haskell. 80 col is plenty wide, particularly if you avoid excessive nesting.

Reply:
If Hernan is OK, sure. I personally never use split screens, so to me it's always just a waste of space.

Reply:
![2021-11-22-142204_2559x1164_scrot](https://user-images.githubusercontent.com/1339296/142877995-b92ac814-5c9c-4df2-8bb6-c192872601c4.png)

This is what my editing layout typically looks like. I wouldn't be able to do this with 3 split panes if the document width was 120. I also use vertical monitors which favors narrower, taller source code.

Reply:
I tend to prefer 120 over 80 but is not a strong opinion. As long as the format is made automatically Im ok with both options. @bwbush any particular opinion on this?

Reply:
The way I think about line width is like I think about responsive web design: desktop-first designs are better on large screens, but unusable on small ones. mobile-first designs work well on all screen sizes, large and small.

Similarly, vertical scrolling is relatively easy to manage. Horizontal scrolling is a nuisance, as our editors and tools are all optimized for vertical navigation first. If you have a large editor screen, covering more horizontal space might allow you to fit more code on screen, but it makes the code much less navigable for those of us who prefer narrow windows, use split panes, etc. Narrower and taller layouts are perfectly usable by everyone. If you have large amounts of unused screen real estate as a result, you could use more space for other panels in your IDE, or tile a second window like a web browser side-by-side with your editor for example.